### ISSUE TITLE
Export function to Excel 

### ISSUE THREAD

**Issue by Ranf16** [icra/ecam#431]
@ElaChe00 
If the implementation is not too complicated and there is still room for further development I would consider the following:

Users that will use the Excel to collect input data might import it into ECAM and modify some data but then they want to collect data for additional substages etc. For collection the additional data they might prefer to use the excel tool (or the user of ECAM is different than the person that provides the data). 

Therefore it would be useful to be able to export the excel file with the inputs already in use in order to modify or add more inputs to it before importing it again.

**Comment by holalluis** [comment]
I agree that this would be very useful, I note it for next iteration
